pages/about.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Summit</title>
+    <link rel="stylesheet" href="./aboutStyle.css">
+</head>
+
+<body>
+    <object data="navBar.html" class="header"></object>
+
+    <main>
+        <section id="about">
+            <!-- Hero section content -->
+            <div class="welcome-msg">
+                <span class="msg1">"Greetings, Tech Enthusiasts!"</span>
+                <h1>
+                    <span class="bold"><i class="fa-sharp fa-solid fa-shield-halved"></i></span>Secure the Future: <span
+                        class="bold">Optimus Cage
+                        Defender</span> Cybersecurity Summit 2023<span class="bold"><i
+                            class="fa-sharp fa-solid fa-shield-halved"></i></span>
+                </h1>
+                <div class="msg2">
+                    Welcome to the "Secure the Future: Optimus Cage Defender Cybersecurity Summit 2023," a prestigious
+                    gathering of
+                    cybersecurity experts and technology enthusiasts from around the world. This groundbreaking
+                    conference focuses on
+                    fortifying digital environments and safeguarding invaluable assets. Explore keynote presentations,
+                    interactive
+                    workshops, and thought-provoking sessions covering network security, threat intelligence, and more.
+                    Connect with
+                    professionals, leaders, and potential collaborators, fostering partnerships and exponential growth.
+                    Whether you're a
+                    cybersecurity aficionado, organizational luminary, or tech enthusiast, this summit offers a unique
+                    opportunity to expand
+                    knowledge and contribute to a resilient digital landscape. Join us on this extraordinary journey to
+                    shape the future of
+                    cybersecurity.
+                </div>
+                <div class="contact">
+                    <div class="p">Please contact us per Email for any further questions about <span class="bold">Otimus
+                            Cage Defender</span>
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, I have noticed that the about page has a page on top of it which is not professional 😨, please try to address this issue for the best user experience 🙏 

https://user-images.githubusercontent.com/44798044/240849599-b670c06a-763d-4e1f-9e2a-ce02cc1a8fb3.mp4